src/syscheckd/db/include/db.hpp

```diff
@@ -8,1073 +8,48 @@
 
 #ifndef FIMDB_H
 #define FIMDB_H
+#include "commonDefs.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+// #define fim_db_decode_registry_value_full_row(stmt) _fim_db_decode_registry_value(stmt, 11)
```

Comment:
This define will be used in the future? Now, it doesn't appear in the rest of the code. If not, let's remove it. Otherwise, let's include a descriptive comment to review it later.

Reply:
This define is to decode the results of a sqlite3 query to a valid FIM structure.
This needs to be removed